website/static/js/osfHelpers.js

```diff
@@ -390,6 +390,35 @@ var htmlEscape = function(text) {
     return $('<div/>').text(text).html();
 };
 
+/**
+ * Resize table to match thead and tbody column
+ */
+
+var tableResize = function(selector, checker) {
```

Comment:
We talked with Nan about this. I know this is not an elegant or efficient solution, especially with the resize listener but I don't think there are many non-js options. The problem is that only the table body gets scrolling in this case and browser scrollbar positioning differences makes it hard to plan for a specific scenario. 

The html alternative would be to separate the thead from table and and scroll an outer layer of the table so the tbody never actually gets a scrollbar and absolute position the new table head. This is also not as elegant but reduces js load.